site/fastdp/fastdp-how-it-works.md

```diff
@@ -0,0 +1,24 @@
+---
+title: How Fast Datapath Works
+layout: default
+---
+
+
+Weave implements an overlay network between Docker hosts, where each packet is encapsulated in a tunnel protocol header and then sent to the destination host, where the header is removed. The Weave router is a user space process, which means that the packet follows a winding path in and out of the Linux kernel:
```

Comment:
This seems to be a description of the non-fast datapath, which either needs to be clearly noted here or moved somewhere else.